tests/bin/eslint.js

```diff
@@ -122,21 +122,35 @@ describe("bin/eslint.js", () => {
             return assertExitCode(child, 1);
         });
 
-        it("gives a detailed error message if no config file is found in / or in ~/", () => {
-
-            // Assumes the root and home directories have no .eslintrc file.
-            const child = runESLint(["--stdin"], { cwd: "/" });
-
-            const exitCodePromise = assertExitCode(child, 1);
-            const stdoutPromise = getOutput(child).then(output => {
-                assert.match(output.stderr, /ESLint couldn't find a configuration file/);
-            });
+        it(
+            "gives a detailed error message if no config file is found in /",
```

Comment:
As a result of https://github.com/eslint/eslint/commit/f7ed84f5eaa9963f4d55709f167d1deed0424e28, the documentation now tells the user to delete any config files from their home directory. However, with this change applied, that advice is no longer necessary. Can you update the documentation to remove that note?

Reply:
Done.

Reply:
Oops. Yes, indeed. Will do.
Done.